### PR TITLE
Merge rocket updates, gRPC samples, jupyter notebooks

### PR DESCRIPTION
**Pull request by naiteeks**

### CODE REVIEW

**Comment by naiteeks** [comment]
@gadamilan , @avkewalr-msft , @anilmur  - please review and merge

File: ref-apps/readme.md [hunk 1]
```diff
@@ -1,13 +1 @@
 # Reference apps
-
-This folder has LVA edge related content
```
Comment (avkewalr-msft) [inline ref-apps/readme.md:3]: @naiteeks - are we removing Iot central stuff from this check in?

Comment (naiteeks) [reply ref-apps/readme.md:3]: no we are not... i think just the readme was updated... i think the rationale was to add a better description for rocket and iot central.. we can add that later...

Comment (naiteeks) [reply ref-apps/readme.md:3]: if we can add rocket information, then I can update this file.

Comment (naiteeks) [reply ref-apps/readme.md:3]: Updated the filw with the links.  @avkewalr-msft /; can you review?

**Comment by avkewalr-msft** [comment]
@gadamilan, @naiteeks -  looks good to go

**Pull request merged**